Automattic/vip-go-mu-plugins
# Fixes attachments with completely missing meta

If an attachment exists, but has no attachment meta, we should still add the filesize.  This is already set up to happen in `Meta_Updater::update_attachment_filesize()` but the check in `Meta_Updater::get_attachments()` does not allow the attachment ID to be fixed if it does not already have attachment metadata.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Delete an attachment meta, or find an attachment that is missing meta
1. Run the WP-CLI command to fix missing filesizes
1. Verify filesizes are added to attachment meta.
